### PR TITLE
docs: implement getting started site

### PR DESCRIPTION
**Pull request by ducpm511**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 [vc-kit doc site](https://uncefact.github.io/vckit/)
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: documentation/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "vckit-documentation",
+  "version": "0.0.0",
```
Comment (ldhyen99) [inline documentation/package.json:3]: I think we should put a version starting with 1, such as 0.0.1 or 1.0.0.

File: documentation/src/pages/markdown-page.md
Comment (ldhyen99) [inline documentation/src/pages/markdown-page.md:1]: I am curious why we need this md file here.

Comment (ducpm511) [reply documentation/src/pages/markdown-page.md:1]: yeah this should be removed, it's one of the sample files generated when I initialized the project

File: documentation/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+FROM base as development
+WORKDIR /home/node/app
+# Install (not ci) with dependencies, and for Linux vs. Linux Musl (which we use for -alpine)
+RUN npm install
+# Switch to the node user vs. root
+USER node
+# Expose port 3000
+EXPOSE 3000
+# Start the app in debug mode so we can attach the debugger
+CMD ["npm", "start"]
```
Comment (namhoang1604) [inline documentation/Dockerfile:25]: I think we don't need it for the dev environment. On dev, the developer can start the server easily on the local machine

**Review by namhoang1604 — APPROVED** [review]

**Pull request merged**